common/CommonAPI.h

```diff
@@ -495,8 +495,14 @@ class CommonAPI
 			params.windowCb->setInputSystem(nbl::core::smart_refctd_ptr(result.inputSystem));
 			if (!params.window)
 			{
-				result.windowManager = nbl::core::make_smart_refctd_ptr<nbl::ui::CWindowManagerWin32>(); // on the Android path
```

Comment:
@sadiuk what was this comment about?

Reply:
or @Crisspl 

I remember that Android was weird cause it fave you an already created window and there was nothing you could do about it + the initialization was split into several weird steps